src/EditorFeatures/Core/Shared/Options/FeatureOnOffOptions.cs

```diff
@@ -56,20 +56,84 @@ internal static class FeatureOnOffOptions
+            storageLocations: new RoamingProfileStorageLocation("TextEditor.%LANGUAGE%.Fix Add Remove Braces"));
+
+        public static readonly PerLanguageOption<bool> FixAccessibilityModifiers = new PerLanguageOption<bool>(
+            nameof(FeatureOnOffOptions), nameof(FixAccessibilityModifiers), defaultValue: true,
+            storageLocations: new RoamingProfileStorageLocation("TextEditor.%LANGUAGE%.Fix Accessibility Modifiers"));
+
+        public static readonly PerLanguageOption<bool> SortAccessibilityModifiers = new PerLanguageOption<bool>(
+            nameof(FeatureOnOffOptions), nameof(SortAccessibilityModifiers), defaultValue: true,
+            storageLocations: new RoamingProfileStorageLocation("TextEditor.%LANGUAGE%.Sort Accessibility Modifiers"));
+
+        public static readonly PerLanguageOption<bool> MakeReadonly = new PerLanguageOption<bool>(
+            nameof(FeatureOnOffOptions), nameof(MakeReadonly), defaultValue: false,
+            storageLocations: new RoamingProfileStorageLocation("TextEditor.%LANGUAGE%.Make Readonly"));
+
+        public static readonly PerLanguageOption<bool> RemoveUnnecessaryCasts = new PerLanguageOption<bool>(
+            nameof(FeatureOnOffOptions), nameof(RemoveUnnecessaryCasts), defaultValue: false,
+            storageLocations: new RoamingProfileStorageLocation("TextEditor.%LANGUAGE%.Remove Unnecessary Casts"));
+
+        public static readonly PerLanguageOption<bool> FixExpressionBodiedMembers = new PerLanguageOption<bool>(
+            nameof(FeatureOnOffOptions), nameof(FixExpressionBodiedMembers), defaultValue: false,
+            storageLocations: new RoamingProfileStorageLocation("TextEditor.%LANGUAGE%.Fix Expression Bodied Members"));
+
+        public static readonly PerLanguageOption<bool> FixInlineVariableDeclarations = new PerLanguageOption<bool>(
+            nameof(FeatureOnOffOptions), nameof(FixInlineVariableDeclarations), defaultValue: false,
+            storageLocations: new RoamingProfileStorageLocation("TextEditor.%LANGUAGE%.Fix Inline Variable Declarations"));
+
+        public static readonly PerLanguageOption<bool> RemoveUnusedVariables = new PerLanguageOption<bool>(
+            nameof(FeatureOnOffOptions), nameof(RemoveUnusedVariables), defaultValue: false,
+            storageLocations: new RoamingProfileStorageLocation("TextEditor.%LANGUAGE%.Remove Unused Variables"));
+
+        public static readonly PerLanguageOption<bool> FixObjectCollectionInitialization = new PerLanguageOption<bool>(
+            nameof(FeatureOnOffOptions), nameof(FixObjectCollectionInitialization), defaultValue: false,
+            storageLocations: new RoamingProfileStorageLocation("TextEditor.%LANGUAGE%.Fix Object Collection Initialization"));
+
+        public static readonly PerLanguageOption<bool> FixLanguageFeatures = new PerLanguageOption<bool>(
+            nameof(FeatureOnOffOptions), nameof(FixLanguageFeatures), defaultValue: false,
```

Comment:
what was the process used to determine which features are true by default and which are false?